### ISSUE TITLE
Bad aria-label values for sidebar nav

### ISSUE THREAD

**Issue by terrill** [uweb/uw-2014#140]
A couple of quick-fix accessibility issues: 

- **div#nav_menu-2** has role="navigation" and aria-label="sidebar_navigation". The value of aria-label should be "Sidebar Navigation" (no underscore). Screen readers announce this label to users, so it should be expressed in plain English.
- **nav#desktop-relative** has aria-label="mobile menu that is not visible in the desktop version". This is too much information. Also, is this nav element even used? On my iPhone, I'm just seeing div#nav_menu-2. Same is true if I emulate various mobile devices using Chrome Developer Tools - nav#desktop-relative always seems to be empty. If it's not used, it should be deleted. If it's used in some contexts, presumably to replace the default sidebar menu, it should probably  have aria-label="Sidebar menu", as long as there's only one sidebar menu.

**Comment by lcaple** [comment]
@terrill thank you for these! I'll get them in our bug list.

`nav#desktop-relative` is used, but is only populated on pages using WordPress to automatically generate sidebar menus. It's aria description is actually inaccurate: that menu is visible both on desktop and mobile. I think the label is mixed up with the mobile menu. I'll get this adjusted as well.

**Comment by terrill** [comment]
Thanks @lcaple - I developed a better understanding of how these are used after I submitted my original issue. I've just edited that with new details, including a proposed solution.

**Comment by terrill** [comment]
In the new theme, I'm still seeing the first of the original problems I reported, but the second one has changed. Here's the current state: 

- The `<nav>` that contains the main navigation has no aria-label. It should have something like `aria-label="Main menu"`. 
- The entire banner (`header#masthead`) contains a `<nav>` as its sole child. This creates an extra level that screen reader users need to navigate. Users expect there to be links in a banner, so it doesn't need a `<nav>`. That should be changed to a `<div>`. If that's potentially problematic, a simpler solution that will only affect screen reader users is to add `role="presentation"` to the `<nav>`, which removes its semantics.
- nav#desktop-relative still has **aria-label="mobile menu that is not visible in the desktop version"** (observed on a desktop)

**Comment by lcaple** [comment]
Is this update related to the New theme, or the old theme? I want to make sure to copy it to the correct queue if this is based on the new theme.

**Comment by terrill** [comment]
The issue I originally documented here was related to the old theme - I just now confirmed that those problems still exist. 

My most recent comment applies to the new theme.
